src/store/filter/filter-stat-unit.store.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseAsyncDataStore } from '@store/common'
+import { TCondition } from '@service-providers/common'
+import {
+  filteringProvider,
+  IStatunits,
+} from '@service-providers/filtering-regime'
+
+export type TFilterStatUnitQuery = {
+  datasetName: string
+  conditions: TCondition[]
+  units: string[] | TCondition
+}
+
+export class FilterStatUnitStore extends BaseAsyncDataStore<
+  IStatunits,
+  TFilterStatUnitQuery
+> {
+  constructor() {
+    super()
+  }
+
+  protected fetch(query: TFilterStatUnitQuery): Promise<IStatunits> {
+    return filteringProvider.getStatUnits({
+      ds: query.datasetName,
+      rq_id: String(Date.now()),
```

Comment:
As i remember, `rq_id` is not necessary

Reply:
Yes, but in practice, it  can be an empty string. And also:
```
rq_id argument should be get from result of initiation request
```
But we don't use initiation request. And i think it will be more honest to use empty string.

Reply:
Ok